airflow/cli/commands/plugins_command.py

```diff
@@ -15,17 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 import inspect
-from typing import Any, List, Optional, Union
-
-from rich.console import Console
+from typing import Any, Dict, List, Union
 
 from airflow import plugins_manager
-from airflow.cli.simple_table import SimpleTable
-from airflow.configuration import conf
+from airflow.cli.simple_table import AirflowConsole
 from airflow.plugins_manager import PluginsDirectorySource
+from airflow.utils.cli import suppress_logs_and_warning
 
 # list to maintain the order of items.
```

Comment:
This line of comment can be removed if we confirm `PLUGINS_MANAGER_ATTRIBUTES_TO_DUMP` can be removed